kubeadm/kube-proxy/build.ps1

```diff
@@ -0,0 +1,71 @@
+param(
+    [string]$image = "sigwindowstools/kube-proxy",
+    [switch]$push
+)
+
+[int]$minMajor = 1
+[int]$minMinor = 17
+[int]$minBuild = 0
+
+$env:DOCKER_CLI_EXPERIMENTAL = "enabled"
+& docker buildx create --name img-builder --use
+
+$output="docker"
+if ($push.IsPresent) {
+    $output="registry"
+}
+
+
+function buildKubeProxy([string]$tag) 
```

Comment:
This looks the same as flannel minus image name can this be made re-usable across flannel and kubeproxy?

Reply:
Maybe it is possible. I don't think about it because this code has some differents.

Reply:
I have done it.